int32 scan1_index                     # Index of scan1
int32 scan2_index                     # Index of scan2
float32[] X0                          # Initial estimte of rigd transform X 
sensor_msgs/PointCloud2 point_cloud1  # Point cloud 1
sensor_msgs/PointCloud2 point_cloud2  # Point cloud 2
